react/features/chat/actions.web.js

```diff
@@ -27,6 +28,27 @@ export function openChat(participant: Object) {
     };
 }
 
+/**
+ * Displays the chat panel for a participant identified by an id.
+ *
+ * @param {string} id - The id of the participant.
+ * @returns {{
+ *     participant: Participant,
+ *     type: OPEN_CHAT
+ * }}
+ */
+export function openChatById(id: string) {
+    return function(dispatch: (Object) => Object, getState: Function) {
+        const participant = getParticipantById(getState(), id);
+
+        return dispatch({
```

Comment:
if this requires the participant object, why not pass it directly in this function? seems like it's available in the calling component